### DISCUSSION TITLE
Request: Support Multiple Authenticator Apps

### DISCUSSION THREAD

**Discussion by johnaweiss** [community/community#40710] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Other online systems support multiple authenticator apps, but not github. 

I use an auth app on my phone. If my phone is dead or broken or lost, i can't login. 

I also use a browser-based authenticator. If i i'm using a different computer, or have to re-install my browser, i may lose the auth, and then can't login. 

Yes, i know i have the backup numbers, but if my computer dies (where the backup numbers are stored) and i have to use some other computer, i can't login. Sure i could keep backups of the backups, but it's getting complicated. 

It will really make my github more convenient, and less risky, if you would support multiple authenticator apps.

**Comment by ossanna16** [comment]
Hi there @johnaweiss and welcome to our community! Thank you for the feedback!
